packages/flutter/lib/src/cupertino/route.dart

```diff
@@ -346,20 +357,22 @@ class CupertinoPageTransition extends StatelessWidget {
     @required bool linearTransition,
   }) : assert(linearTransition != null),
        _primaryPositionAnimation = (linearTransition ? primaryRouteAnimation :
+       // The curves below have been rigorously derived from plots of native
```

Comment:
As they get more and more certain, be more specific (i.e. stop frames filmed then measured on screen)